kolibri/plugins/coach/assets/src/vue/index.vue

```diff
@@ -38,40 +38,40 @@
     },
     components: {
       'top-nav': require('./top-nav'),
-      'class-list-page': require('./class-list-page'),
-      'recent-page': require('./recent-page'),
-      'topics-page': require('./topics-page'),
-      'exams-page': require('./exams-page'),
-      'learners-page': require('./learners-page'),
-      'groups-page': require('./groups-page'),
+      'class-list': require('./class-list'),
+      'recent': require('./recent'),
+      'topics': require('./topics'),
+      'exams': require('./exams'),
+      'learners': require('./learners'),
+      'groups': require('./groups'),
```

Comment:
For component names, I'd prefer to leave on the `-page` suffix. (This helps distinguish top-level components that have a URL from other, shared components)